data/binding/gen.go

```diff
@@ -33,8 +33,7 @@ type External{{ .Name }} interface {
 //
 // Since: {{ .Since }}
 func New{{ .Name }}() {{ .Name }} {
-	var blank {{ .Type }}
-	blank = {{ .Default }}
+	var blank {{ .Type }} = {{ .Default }}
```

Comment:
I think this assignment is unneeded, should be enough this:
```go
var blank {{ .Type }}
```

Reply:
It depends on what the value of `.Default` is - working without it may be coincidence.
We'd be ignoring the default field, which doesn't seem right.

Reply:
I guess what I was meaning is whether that would be a different change. This PR was just adding a new type.

Reply:
> I guess what I was meaning is whether that would be a different change. This PR was just adding a new type.

Oh sorry, I just thought you wanted to fix this too, because you modified it. Then it is ok, sorry for confusion.